### PR TITLE
Add ATtiny13A

### PR DESCRIPTION
**Pull request by sanbeg**
##### SUMMARY

Copy/modify existing ATtiny85 entries to create similar Attiny13a

##### OPEN QUESTIONS / UNRESOLVED ISSUES

How do the pin labels get rendered?  Although this is copied from the ATtiny85, currently only VCC & GND are labeled; other labels don't display.

##### CHECKLIST

- [x] I have read and followed the [library conventions](https://docs.librepcb.org/#libraryconventions).
- [x] For packages, I followed IPC7351C (see details in library conventions).
- [x] I'm the copyright owner of the added content (i.e. the changes are made by myself, not copied/imported from somewhere else).
- [x] I agree to publish all my changes under the [CC0 Public Domain License](https://en.wikipedia.org/wiki/CC0), allowing everyone to use and modify the content without any restrictions.

### CODE REVIEW

**Comment by dbrgn** [comment]
Thanks, the ATtiny13A is a great chip.

The symbol of the ATtiny13 and the ATtiny85 are identical, so we could reuse them. Could you rename the "ATtiny85" symbol to "ATtiny 8-Pin", make the description more generic and then use that symbol for the component?

Furthermore, I think the official spelling is `ATtiny13A` (with capital A). Could you adjust that?

> How do the pin labels get rendered? Although this is copied from the ATtiny85, currently only VCC & GND are labeled; other labels don't display.

The reason is that the pin-signal-mappings in the component are not complete. If you open the component and edit the variant under "Symbol Variants" (by clicking the edit icon), you can see that pins PB0-5 are not assigned to a signal. You need to do that manually.

File: cmp/72a8f329-a505-4c99-9967-86e29558828a/component.lp [hunk 1]
```diff
@@ -1,9 +1,9 @@
 (librepcb_component 72a8f329-a505-4c99-9967-86e29558828a
  (name "ATtiny85")
- (description "Atmel 8-bit AVR Microcontroller with 8 bytes in-system programmable flash.")
+ (description "Atmel 8-bit AVR Microcontroller with 8 kilobytes in-system programmable flash.")
  (keywords "attiny,atmel,avr,8bit")
  (author "Danilo Bargen")
- (version "0.1")
+ (version "0.2")
```
Comment (dbrgn) [inline cmp/72a8f329-a505-4c99-9967-86e29558828a/component.lp]: No need to bump the version here for a description change.

```suggestion
 (version "0.1")
```

File: cmp/74525bd1-3387-4bd6-ab0d-ed3969b6048b/component.lp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ )
+ (signal 1aa108ab-bbfc-4fa5-9862-324acad0b10f (name "GND") (role passive)
+  (required true) (negated false) (clock false) (forced_net "")
+ )
+ (signal a0dc9e3a-c2a9-4389-b1e9-261be6d9f70f (name "PB0/MOSI/AREF/OC0A/PCINT0") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 9264c42b-76c4-4b0a-8ac4-1204371363e8 (name "PB1/MISO/OC0B/INT0/PCINT1") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
```
Comment (dbrgn) [inline cmp/74525bd1-3387-4bd6-ab0d-ed3969b6048b/component.lp]: Until the issue with the overlapping pin labels is resolved (see #11), I'd skip the counter/timer functionality:

```suggestion
 (signal a0dc9e3a-c2a9-4389-b1e9-261be6d9f70f (name "PB0/MOSI/AREF/PCINT0") (role passive)
  (required false) (negated false) (clock false) (forced_net "")
 )
 (signal 9264c42b-76c4-4b0a-8ac4-1204371363e8 (name "PB1/MISO/INT0/PCINT1") (role passive)
  (required false) (negated false) (clock false) (forced_net "")
 )
```

That should avoid the issue for now.

**Review by dbrgn — CHANGES_REQUESTED** [review]
Thank you again, and sorry for the long delay in reviewing!

Two remaining minor change requests, then it should be ready to merge!

**Comment by ubruhin** [comment]
Sorry for the long delay on this PR. I reviewed it now and fixed some remaining issues to finally get it merged:

- Renamed device "ATtiny13a-20Px" to "ATtiny13A-PU" to reflect a real, orderable device. If somebody needs a slightly different model (e.g. the older ATtiny13-20PU), a separate device should be added to the library.
- Renamed device "ATtiny13a-20Sx" to "ATtiny13A-SU" for the same reason.
- The device "ATtiny13a-20Sx" had the wrong package assigned (wrong height), thus I changed it from SOIC127P762X140-8 to SOIC127P762X**170**-8. Since this is a breaking change, I assigned a new UUID to this device. So basically the old device has been removed and a new one added with the correct package.
- Removed OC0A and OC0B from component signal names due to limited space in the symbol.

I hope these changes are fine for you.

**Pull request merged**